tensorflow/python/framework/tensor_util.py

```diff
@@ -122,39 +122,39 @@ def FastAppendBFloat16ArrayToTensorProto(tensor_proto, proto_values):
 else:
 
   def SlowAppendFloat32ArrayToTensorProto(tensor_proto, proto_values):
-    tensor_proto.float_val.extend([np.asscalar(x) for x in proto_values])
+    tensor_proto.float_val.extend([x.item() for x in proto_values])
 
   def SlowAppendFloat64ArrayToTensorProto(tensor_proto, proto_values):
-    tensor_proto.double_val.extend([np.asscalar(x) for x in proto_values])
+    tensor_proto.double_val.extend([x.item() for x in proto_values])
```

Comment:
What do you think about replacing `[x.item() for x in proto_values]` with `proto_values.tolist()`?

I'm not very familiar with this part of the code base, but it looks like [`numpy.ndarray.tolist()`](https://www.numpy.org/devdocs/reference/generated/numpy.ndarray.tolist.html) uses `numpy.ndarray.item()` underneath.